src/ios/PSPDFKitPlugin.m

```diff
@@ -1970,4 +1975,33 @@ - (void)processAnnotations:(CDVInvokedUrlCommand *)command {
     }
 }
 
+#pragma mark - Notifications
+
+- (void)annotationChangedNotification:(NSNotification *)notification {
+    id object = notification.object;
+    NSArray <PSPDFAnnotation *> *annotations;
+    if ([object isKindOfClass:NSArray.class]) {
+        annotations = object;
+    } else if ([object isKindOfClass:PSPDFAnnotation.class]) {
+        annotations = @[object];
+    } else {
+        return;
+    }
+
+    NSString *name = notification.name;
+    NSString *changeEventName;
+    if ([name isEqualToString:PSPDFAnnotationChangedNotification]) {
+        changeEventName = @"onAnnotationChanged";
```

Comment:
Should we use plural since we provide the object in an `annotations` array?
I know that this will ever only be a single annotation for now, but might make it more obvious and also future-proof if this might change someday.

```suggestion
        changeEventName = @"onAnnotationsChanged";
```

Reply:
Great catch! Thank you, @steviki!